### DISCUSSION TITLE
Deployment of Trino on Ambari 2.7.5 (Bigtop 3.2.0)

### DISCUSSION THREAD

**Discussion by yassiracharki** [community/community#51240] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hi Everyone, i have a working Hadoop Cluster running Bigtop 3.2.0 on Ambari 2.7.5, I want to add Trino to the cluster, is there any specific documentation for this use-case ?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @yassiracharki ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by yassiracharki** [reply]
thanks, i hope someone sees it now (:
